core/store/tx_manager.go

```diff
@@ -364,24 +363,24 @@ func isUnderPricedReplacementError(err error) bool {
 }
 
 // SignedRawTxWithBumpedGas takes a transaction and generates a new signed TX from it with the provided params
-func (txm *EthTxManager) SignedRawTxWithBumpedGas(originalTx models.Tx, gasLimit uint64, gasPrice big.Int) (string, error) {
+func (txm *EthTxManager) SignedRawTxWithBumpedGas(originalTx models.Tx, gasLimit uint64, gasPrice big.Int) (slice []byte, err error) {
 	ma := txm.getAccount(originalTx.From)
+	rlp := new(bytes.Buffer)
 	if ma == nil {
-		return "", fmt.Errorf("Unable to locate %v as an available account in EthTxManager. Has TxManager been started or has the address been removed?", originalTx.From.Hex())
+		return slice, fmt.Errorf("Unable to locate %v as an available account in EthTxManager. Has TxManager been started or has the address been removed?", originalTx.From.Hex())
 	}
 
 	transaction := types.NewTransaction(originalTx.Nonce, originalTx.To, originalTx.Value.ToInt(), gasLimit, &gasPrice, originalTx.Data)
 
-	transaction, err := txm.keyStore.SignTx(ma.Account, transaction, txm.config.ChainID())
+	transaction, err = txm.keyStore.SignTx(ma.Account, transaction, txm.config.ChainID())
 	if err != nil {
-		return "", err
+		return slice, err
```

Comment:
This one I think I'd be more concerned about -- named return values can cause weird shadowing issues, so it's generally a good idea to avoid them unless absolutely necessary.  A simple `return nil, err` would reduce complexity and footguns here.